vertical-pod-autoscaler/enhancements/3919-customized-recommender-vpa/README.md

```diff
@@ -185,9 +199,24 @@ func NewClusterState(recommender_name string) *ClusterState {
 }
 ```
 
-Therefore, when loading VPA objects to the `clusterStateFeeder`, it can use the field selector to select VPA CRDs that 
-have `recommenderName` equal to the current clusterState’s `RecommenderName`.
+Therefore, when loading VPA objects to the `clusterStateFeeder`, it should
+ignore VPA objects that don't have an item in the `recommenders` field with
+`name` equal to the current clusterState’s `RecommenderName`.
+
 ```golang
+func implicitDefaultRecommender(selectors[]*VerticalPodAutoscalerRecommenderSelector) bool {
+	return len(selectors) == 0
+}
+
+func selectsRecommener(selectors[]*VerticalPodAutoscalerRecommenderSelector, name *string) bool {
```

Comment:
nit: selectsRecommender (missing 'd')?
Or should it be selected ?

Reply:
Done.